src/serac/infrastructure/profiling.hpp

```diff
@@ -55,10 +73,16 @@
 #define SERAC_MARK_LOOP_START(id, name)
 #define SERAC_MARK_LOOP_ITER(id, i)
 #define SERAC_MARK_LOOP_END(id)
+#define SERAC_MARK_START(name)
+#define SERAC_MARK_END(name)
+#define SERAC_SET_METADATA(name, data) 
 
 #endif
 
+// profiling namespace
```

Comment:
Should this be /// or 98 be //?  Seems to be inconsistent

Reply:
Good catch! It just depends on what is necessary to get doxygen to recognize namespace functions.